src/components/utils/src/logger/std_logger.cc

```diff
@@ -0,0 +1,59 @@
+#include "utils/logger/std_logger.h"
```

Comment:
@VladSemenyuk please go over all new added files and insert copyright comments in each of them